--- configure.tgt.original	2018-10-14 16:45:20.851810206 -0400
+++ configure.tgt	2018-10-14 16:55:37.952297703 -0400
@@ -262,6 +262,15 @@
 			targ_extra_emuls=elf_i386 ;;
 i[3-7]86-*-sysv[45]*)	targ_emul=elf_i386
 			targ_extra_emuls=elf_iamcu ;;
+i[3-7]86-*-saturn*)
+			targ_emul=elf_i386_saturn
+			targ_extra_emuls=elf_i386
+			targ64_extra_emuls="elf_x86_64_saturn elf_x86_64"
+			;;
+x86_64-*-saturn*)
+			targ_emul=elf_x86_64_saturn
+			targ_extra_emuls="elf_x86_64 elf_i386_saturn elf_i386"
+			;;
 i[3-7]86-*-solaris2*)	targ_emul=elf_i386_sol2
 			targ_extra_emuls="elf_i386_ldso elf_i386 elf_iamcu elf_x86_64_sol2 elf_x86_64 elf_l1om elf_k1om"
 			targ_extra_libpath=$targ_extra_emuls
@@ -794,6 +803,14 @@
   NATIVE_LIB_DIRS='/usr/local/lib /usr/ccs/lib /lib /usr/lib'
   ;;
 
+i[3-7]86-*-saturn*)
+  NATIVE_LIB_DIRS='/libraries /system/libraries'
+  ;;
+
+x86_64-*-saturn*)
+  NATIVE_LIB_DIRS='/libraries /system/libraries'
+  ;;
+
 i[3-7]86-*-solaris*)
   NATIVE_LIB_DIRS='/usr/local/lib /usr/ccs/lib /lib /usr/lib'
   ;;
